geozero/src/geometry_processor.rs

```diff
@@ -123,21 +126,46 @@ pub trait GeomProcessor {
 
     /// Begin of MultiPoint processing
     ///
-    /// Next: size * xy/coordinate
+    /// Next: `size` calls to [`xy()`][`Self::xy()`] or [`coordinate()`][`Self::coordinate()`]
+    ///
+    /// ## Parameters
+    ///
+    /// - `size`: the number of Points in this MultiPoint
+    /// - `idx`: the positional index of this MultiPoint. This will be 0 except in the case of a
+    ///   GeometryCollection.
+    ///
+    /// ## Following events
+    ///
+    /// - [`point_begin`][`Self::point_begin()`] for each contained point (NOTE: seems expected but seems not implemented)
```

Comment:
I just created https://github.com/georust/geozero/issues/184 with a longer explanation. Can you link to it here in the docs?

Reply:
I removed the mentions of `point_begin` and `point_end` and linked to #184 for discussion